### ISSUE TITLE
redsocks command not found!

### ISSUE THREAD

**Issue by akilaid** [abdoxfox/HTTP-CUSTOM-HEADERS-VPN#31]
I'm having trouble with redsocks 
![image](https://user-images.githubusercontent.com/45641122/233577291-f428717b-4160-4cd4-af2a-fc0b501f255f.png)

**Comment by abdoxfox** [comment]
are you using Ubuntu?

if true, should install redsocks package:
apt install redsocks
Then modify runvp.sh  in order to run with installed one

**Comment by abdoxfox** [comment]
# you can use this modified file

```
# create new chain
iptables -t nat -N PROXY
iptables -t nat -I OUTPUT -j PROXY

# exclude local traffic, see: http://manpages.org/ss-redir
iptables -t nat -A PROXY -d 127.0.0.0/8 -j RETURN
iptables -t nat -A PROXY -d 192.168.0.0/16 -j RETURN
iptables -t nat -A PROXY -d 0.0.0.0/8 -j RETURN
iptables -t nat -A PROXY -d 10.0.0.0/8 -j RETURN
iptables -t nat -A PROXY -d 169.254.0.0/16 -j RETURN
iptables -t nat -A PROXY -d 172.16.0.0/12 -j RETURN
iptables -t nat -A PROXY -d 224.0.0.0/4 -j RETURN
iptables -t nat -A PROXY  -d 240.0.0.0/4 -j RETURN


# socksify whole TCP traffic
iptables -t nat -A PROXY -p tcp -j REDIRECT --to-ports 8123
#iptables -t nat -A PROXY -p tcp -j REDIRECT --to-ports 8124

# socksify only DNS UDP traffic
iptables -t nat -A PROXY -p udp --dport 53 -j REDIRECT --to-ports 5313
iptables -t nat -A PROXY -p udp --dport 53 -j REDIRECT --to-ports 5314
iptables -t nat -A PROXY -p udp --dport 53 -j REDIRECT --to-ports 7300

trap "iptables -t nat -D OUTPUT -j PROXY; iptables -t nat -F PROXY; iptables -t nat -X PROXY" EXIT

# run socksifier

sudo ./redsocks -c redsocks.conf

**Comment by akilaid** [comment]
I found that there is a problem with dnsu2t in redsocks.conf file when I'm using redsocks from apt. then I tried compiling redsocks from your repo. Somehow redsocks seems crashing when I try to do `curl ifconfig.me`. If you can give me the correctly compiled redsocks binary, I can try connect server using it.

Thank you considering about my issue ✌️

**Comment by abdoxfox** [comment]
are you using this repo inside vps?

**Comment by akilaid** [comment]
No, I'm just using putty to connect my Orange Pi Zero via SSH locally.

**Comment by abdoxfox** [comment]
> 

remove dnsu2t section and try again

**Comment by akilaid** [comment]
Actually I have tried it already 🙁

**Comment by abdoxfox** [comment]
Yeah , over sni tunnel but you should mention that your ssh servers should support stunnel

**Comment by akilaid** [comment]
Huge thanks to you! I have managed to connect to my server successfully using regular internet and it is perfectly fine! Now the problem is my secondary connection only has access to google.com with 8.8.8.8, 8.8.4.4 DNS. I think redsocks is not forwarding DNS queries through tunnel.

![image](https://user-images.githubusercontent.com/45641122/233789856-96d4ac28-2727-436a-8535-ac6f047bcff8.png)

**Comment by akilaid** [comment]
Huge thanks to you! I have managed to connect to my server successfully using regular internet and it is perfectly fine! Now the problem is my secondary connection only has access to google.com with 8.8.8.8, 8.8.4.4 DNS. I think redsocks is not forwarding DNS queries through the tunnel.
![Screenshot 2023-04-22 194405](https://user-images.githubusercontent.com/45641122/233789971-e9c15e5d-d5e8-49df-b36a-c1a3030fdbec.png)

**Comment by akilaid** [comment]
I was managed to route DNS throgh using https://github.com/jtripper/dns-tcp-socks-proxy. Huge thanks for the feedback you gave me and making such a nice repository. ✌️

**Comment by abdoxfox** [comment]
could you please contact me at telegram 
My profile link : t.me/Abdoxfox_official
